lib/Window.js

```diff
@@ -95,6 +96,14 @@ module.exports = function Window(resp, windowObjects = {console}, userAgent) {
     return emitter.emit(event.type, event);
   }
 
+  function pushState(ign1, ign2, relativeUrl) {
+    const newUrl = url.parse(relativeUrl);
+    location.path = newUrl.path;
+    location.pathname = newUrl.pathname;
+    location.search = newUrl.search;
+    location.href = url.format(location);
+  }
```

Comment:
hm, identical to `replaceState`, no? if they behave the same why not use the same function?

Reply:
Yes, but since the history object does not store any history the implementations are the same. Maybe some storage for the history should be added to the history object

Reply:
Might be worth just going the extra mile and implement popState with a simple history queue while your'e here.
Otherwise just expose the replaceState as both replaceState and pushState.
Your call!

Reply:
I agree.

Reply:
By `popState` do you mean to implement the `back` functionality of the `history` object? Since the `history` object does not seem to implement any `popState` functionality

Reply:
Oh, I thought there was a `popState`, but yes `back` I guess

Reply:
Now I have added functions for `back`, `forward` and `go` and added unit tests for these functions

Reply:
@jonaswalden Do you wanna check one more time, or can I merge?